### PR TITLE
[WIP] Add coredump to systemd

### PR DESCRIPTION
**Pull request by alexhumphreys**

### CODE REVIEW

**Comment by alexhumphreys** [comment]
CAREFUL:
this will add coredump to systemd. However, I don't think it has python symbols, and it causes a rebuild of practically everything so takes like 2 hours. Not sure if we wanna merge this just yet.

File: meta-jaspar/recipes-core/systemd/systemd_225.bbappend [hunk 1]
```diff
@@ -0,0 +1,5 @@
+EXTRA_OECONF := "${@oe_filter_out('--disable-coredump', '${EXTRA_OECONF}', d)}"
```
Comment (cajun-rat) [inline meta-jaspar/recipes-core/systemd/systemd_225.bbappend]: if you put EXTRA_OECONF_pn_systemd here, does that reduce the rebuild?

**Comment by alexhumphreys** [comment]
Made that change, but it still seems to do a rather large rebuild. At any rate it rebuilds webkitgtk so that's 90 minutes right there. Merge and build over night?

**Comment by cajun-rat** [comment]
Yep

**Pull request merged**